akka-stream/src/main/scala/akka/stream/impl/fusing/Interpreter.scala

```diff
@@ -124,11 +125,12 @@ private[akka] object OneBoundedInterpreter {
 private[akka] class OneBoundedInterpreter(ops: Seq[Stage[_, _]],
                                           onAsyncInput: (AsyncStage[Any, Any, Any], AsyncContext[Any, Any], Any) ⇒ Unit,
                                           materializer: FlowMaterializer,
+                                          attributes: OperationAttributes = OperationAttributes.none,
```

Comment:
I am not convinced about this. Why would the interpreter need to know about attributes? The stages need to know, but I don't like coupling the interpreter now to yet another dependency.